src/cartridges/int_adyen_controllers/cartridge/templates/default/account/payment/adyenpaymentinstrumentdetails.isml

```diff
@@ -3,12 +3,15 @@
 
 <link rel="stylesheet" href="${URLUtils.staticURL('/css/adyenCheckout.css')}" />
 
+<input type="hidden" id="sessionData" value="${pdict.SessionData}"/>
+
 <script>
     var clientKey = "${AdyenHelper.getAdyenClientKey()}";
     var loadingContext = "${AdyenHelper.getLoadingContext()}";
     var environment = "${AdyenHelper.getAdyenEnvironment().toLowerCase()}";
     var locale = "${request.locale}";
     window.Configuration = {clientKey, loadingContext, environment, locale};
+    window.sessionData = JSON.parse(document.getElementById('sessionData').value);
```

Comment:
Should we parse the `pdict.SessionData` directly here? 
For example `JSON.parse('${pdict.SessionData}'`